package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "oas3-chow-chow",
-  "version": "0.17.0",
+  "version": "0.18.0",
```

Comment:
Is this change from running the `yarn changeset` as well?

Reply:
@supertong no, I added this (I did in my other koa-oas3 PR as well). Does changeset manage this when the maintainer publishes? Let me know and I can revert this change. Thanks.

Reply:
Yes, it will be handled by changeset during the publish.

Reply:
@supertong fixed, thanks